README.md

```diff
@@ -117,6 +117,12 @@ Getting stuck is a natural part of development and happens to the best of us. It
 1. **Your team channel** - When you start a project, you will be added to a private team channel. Feel free to ask your questions there! Your team lead may be able to help you through your blocker.
 2. **[bwtc-stackoverflow](https://stackoverflow.com/c/bwtc/questions)** - Did you know we have our own Stack Overflow? Take a look, it has some great quesions and answers. 
 3. **[bwtc-technical-discussions](https://bitwiseindustries.slack.com/archives/CN462BE5D)** - Great place for technical questions that involve some code.
+
+#### Problem Solving
```

Comment:
I think this section would make more sense if it were placed before Communication Channels. Reading the Problem Solving section first reinforces the idea that we should make an attempt on the problem before reaching out for help.